torch/csrc/utils/tensor_flatten.h

```diff
@@ -4,9 +4,16 @@
 #include <torch/csrc/WindowsTorchApiMacro.h>
 #include <ATen/ATen.h>
 #include <utility>
+#include <c10/core/TensorOptions.h>
 
 namespace torch { namespace utils {
 
+inline int64_t type_id(const at::Tensor& tensor) {
```

Comment:
The type here is `int64_t` but everywhere else we use `size_t`? Should we consolidate the type everywhere?